common-rest/src/main/java/org/uniprot/api/rest/download/heartbeat/HeartBeatProducer.java

```diff
@@ -53,6 +54,13 @@ public void createForIds(DownloadJob downloadJob) {
                         long newUpdateCount = downloadJob.getUpdateCount() + 1;
                         downloadJob.setUpdateCount(newUpdateCount);
                         Failsafe.with(retryPolicy)
+                                .onFailure(
+                                        throwable ->
+                                                log.warn(
+                                                        MessageFormat.format(
+                                                                "Job {0} was failed to update in solr phase {1}",
```

Comment:
Can you please update the message to redis heart beat update
Job {0} failed to update in redis count to {1} and failure {2}